screens/Budget/components/missingBudget.tsx

```diff
@@ -20,24 +21,7 @@ const MissingBudget = (
             year: number
         }) => {
 
-    var closetBudget: string | undefined = undefined
-    if (otherBudgets?.budgets.__typename == 'BudgetsSuccess') {
-        otherBudgets.budgets.budgets.forEach(ele => {
-            const monthString = MONTH_TO_NUM_STRING(ele.month.valueOf())
-            if (monthString === null || monthString === undefined) {
-                return
-            }
-            const curr = ele.year + "-" + MONTH_TO_NUM_STRING(ele.month)
-            if (closetBudget === undefined && curr < year + '-' + month) {
-                closetBudget = curr
-            } else if (closetBudget === undefined) {
-                return
-            }
-            else if ((curr > closetBudget && curr < year + '-' + month)) {
-                closetBudget = curr
-            }
-        })
-    }
+    const [closetBudgetId, setClosetBudgetId] = useState<number | undefined>()
 
 
     const [createBudget] = useMutation<CreateBudgetMutation>(CreateBudgetDocument, {
```

Comment:
I would change this de-structuring to `[createBudget, _]` to make it obvious that there is a second value that is returned but not used.